src/tasks/withdraw/traded_tokens.ts

```diff
@@ -1,8 +1,37 @@
+import axios from "axios";
 import { Contract } from "ethers";
 import { HardhatRuntimeEnvironment } from "hardhat/types";
 
 import { BUY_ETH_ADDRESS } from "../../ts";
 
+export const BUFFER_TRADABLE_TOKENS: Promise<string[]> = axios
+  .get(
+    "https://raw.githubusercontent.com/gnosis/cow-dex-solver/main/data/token_list_for_buffer_trading.json",
+  )
+  .then((response) => {
+    const token_list: { symbol: string; decimals: number; address: string }[] =
+      response.data.tokens;
+    return token_list.map((tokenObject) => tokenObject.address);
+  })
+  .catch((err) => {
+    throw Error(`Warning: unable to recover buffer token list, due to: ${err}`);
+  });
+
+export function excludeBufferTradableTokensFromList(
+  tokens: string[],
+  bufferTradableTokenList: string[],
+) {
+  console.log("Excluding buffer tradable from withdraw list...");
+  const tokenListLength = tokens.length;
+  tokens = tokens.filter((token) => !bufferTradableTokenList.includes(token));
```

Comment:
One thing to watch out for is that I think this is doing a case-sensitive compare. So if the token list doesn't use checksummed addresses for some tokens, they might not get filtered out.

Maybe you could build a `Set` (for lookup performance) of the tokens in a specific case:
```ts
const excludedTokens = new Set(bufferTradableTokenList.map((token) => token.toLowerCase()));
// ...
tokens = tokens.filter((token) => !excludedTokens.has(token.toLowerCase()));
```